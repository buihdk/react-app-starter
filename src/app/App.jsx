import React from 'react';
import { hot } from 'react-hot-loader/root';
import I18n from 'i18n-js';

import PacMan from 'src/components/PacMan';

import './App.css'

const App = () => (
  <>
    <h1>{I18n.t('title')}</h1>
    <p>{I18n.t('instruction')}</p>
    <PacMan />
  </>
);

export default hot(App);
