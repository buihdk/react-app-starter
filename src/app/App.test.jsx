import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';

import App from './App';

describe('App', () => {
  const wrapper = shallow(<App />);

  test('renders without crashing', () => {
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
