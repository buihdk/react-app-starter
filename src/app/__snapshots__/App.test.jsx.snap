// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders without crashing 1`] = `
<Fragment>
  <h1>
    React App Starter
  </h1>
  <p>
    Open src/app/App.jsx to start developing your React app.
  </p>
  <PacMan />
</Fragment>
`;
